src/code/utilities/genetics-utils.js

```diff
@@ -471,4 +471,59 @@ export default class GeneticsUtils {
     });
     return {geneStart, chromosomeHeight: species.chromosomesLength[chromosomeName]};
   }
+
+  /**
+   * Given a gene string, returns an object representing the phenotypes expressed by that gene string. For example,
+   * 'a:w,b:w' is converted to {'wings': 'No wings'}.
+   */
+  static convertGeneStringToPhenotype(genes) {
```

Comment:
I'll make @kswenson's changes for now, and leave a TODO to clean this up the next time we need to make ITS changes